homeassistant/components/number/const.py

```diff
@@ -34,6 +34,7 @@
     UnitOfTemperature,
     UnitOfTime,
     UnitOfVolume,
+    UnitOfVolumeFlowRate,
```

Comment:
Should we name it `UnitOfVolumetricFlowRate` https://en.wikipedia.org/wiki/Volumetric_flow_rate ?

Reply:
It's already named. So it would be a breaking change to rename the type.